cmd/influx_inspect/main.go

```diff
@@ -3,6 +3,7 @@ package main
 
 import (
 	"fmt"
+	"github.com/influxdata/influxdb/cmd/influx_inspect/reportdisk"
```

Comment:
nit: this could move down to the other related imports (`goimports`) can help with this.